### PR TITLE
IBX-902: Fixed field groups in content on the fly edit

### PR DESCRIPTION
**Pull request by damianz5**
| Question      | Answer
| ------------- | ---
| Tickets       | https://issues.ibexa.co/browse/IBX-902
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Tests pass?   | yes
| Doc needed?   | no
| License       | [GPL-2.0](https://github.com/ezsystems/ezplatform-admin-ui/blob/master/LICENSE)

Fixed field groups in content on the fly edit

requires: https://github.com/ezsystems/ezplatform-content-forms/pull/46

#### Checklist:
- [x] Coding standards (`$ composer fix-cs`)
- [x] Ready for Code Review

### CODE REVIEW

File: src/bundle/Controller/ContentOnTheFlyController.php [hunk 1]
```diff
@@ -354,6 +363,9 @@ private function buildEditView(
             'language' => $language,
             'content_type' => $contentType,
             'form' => $form->createView(),
+            'grouped_fields' => $this->groupedContentFormFieldsProvider->getGroupedFields(
+                $form->get('fieldsData')->all()
+            ),
```
Comment (Steveb-p) [inline src/bundle/Controller/ContentOnTheFlyController.php:368]: My only note is here, that field grouping should be done in the form type itself, during `buildView` method.

**Review by Steveb-p — APPROVED** [review]

**Review by webhdx — APPROVED** [review]

**Review by ciastektk — APPROVED** [review]

**Review by mikadamczyk — APPROVED** [review]

**Review by adamwojs — APPROVED** [review]

**Comment by tomaszszopinski** [comment]
Successful Travis build: https://app.travis-ci.com/github/ezsystems/ezplatform-admin-ui/jobs/536619485

**Review by tomaszszopinski — APPROVED** [review]
QA approved on Ibexa DXP 3.3.x-dev with diff.

**Comment by Nattfarinn** [comment]
@damianz5 Please merge up to `master` and do fast-forward merge to `ibexa` org.

**Pull request merged**